ui/src/features/stage/promotion-details-modal.tsx

```diff
@@ -188,6 +234,22 @@ export const PromotionDetailsModal = ({
       onOk={hide}
       onCancel={hide}
       cancelButtonProps={{ hidden: true }}
+      footer={
+        canAbortPromotion(promotion) && (
+          <>
+            <Button onClick={hide} type='primary'>
+              Close
+            </Button>
+            <Button
+              danger
+              icon={<FontAwesomeIcon icon={faStopCircle} className='text-lg' />}
+              onClick={confirmAbortRequest}
+            >
+              Abort
+            </Button>
+          </>
+        )
```

Comment:
I think Close button should always be visible

Reply:
Done. Thanks